towhee/runtime/operator_manager/operator_action.py

```diff
@@ -11,6 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from towhee.runtime.dag_repr import DAGRepr
```

Comment:
Can you remove the dependency of DAGRepr.

Reply:
Done.